inception-kb/src/main/resources/de/tudarmstadt/ukp/inception/kb/knowledgebase-profiles.yaml

```diff
@@ -11,6 +11,7 @@ british_museum:
       property-type: http://www.w3.org/1999/02/22-rdf-syntax-ns#Property
       property-label: http://www.w3.org/2000/01/rdf-schema#label
       property-description: http://www.w3.org/2000/01/rdf-schema#comment
+      base-concept : http://www.w3.org/2002/07/owl#Nothing
```

Comment:
The root concepts in the KB should be added here:

```
root-concepts:
- http://www.w3.org/2002/07/owl#Nothing
```

Reply:
http://www.w3.org/2002/07/owl#Nothing is taken as default (for now) with KBs without any base concept specified yet. We need to get the root concepts for each one of them .

Reply:
@naveen2507 did you try defining a root IRI for Wikidata (official as well as UKP Virtuoso)?